tests/languages/markdown/title_feature.test

```diff
@@ -4,6 +4,9 @@ title 1
 foobar
 ======
 
+Überschrift
+===========

```

Comment:
Can we add a test for the example in your PR comment?

```
Überschrift
-----------
```

Reply:
That, I cannot do because of the way `.test` files are parsed, so I added a similar test.

Reply:
Fair enough. Looks like we had a similar test already but no harm in another.